modules/gob-examine-student/source/compute.ts

```diff
@@ -43,7 +42,7 @@ export default function compute(
       if (isRequirementName(name)) {
         // Primarily for the math major: if a requirement is set to 'children share courses',
         // then they share courses. The default is false (well, undefined).
-        // If they don't share courses, then they share the dirty set;
+        // If they don't share courses, then they share the dirty set,
```

Comment:
```suggestion
        // If they don't share courses, then they share the dirty set;
```

No reason to change this comment